packages/devtools_app/lib/src/main.dart

```diff
@@ -233,7 +234,8 @@ class HtmlPerfToolFramework extends HtmlFramework {
           : notRunningFlutterMsg,
     ));
     addScreen(HtmlTimelineScreen(
-      enabled: isFlutterApp && !isFlutterWebApp,
+      isDartCliApp ? TimelineMode.full : TimelineMode.frameBased,
+      enabled: (isFlutterApp && !isFlutterWebApp) || isDartCliApp,
```

Comment:
this enabled case can probably be simplified to `app.isRunningOnDartVM`

Reply:
done.